lib/render.js

```diff
@@ -69,11 +69,21 @@ function makeState(data, meta) {
     indices = getIndices(meta._layoutRef || meta._ref, data),
     componentList = indices && indices.components || [];
 
+  let layoutName;
 
   // Make an affordance for _self, which was conistently implemented in Amphora pre-v7.
   initialState._self = meta._ref;
   initialState._components = componentList;
 
+  if (meta._layoutRef) {
+    layoutName = clayUtils.getLayoutName(meta._layoutRef);
+    initialState._layout = { name: layoutName, uri: meta._layoutRef };
```

Comment:
I think calling the key in state `_layout` will cause issues downstream when the root partial is invoke with state and composed data.  In the case of a render which includes a layout then the composed data will include a `_layout` property which will overwrite the one being set here because lodash assign is destructive to the first input.  This then causes the media service to not retrieve CSS for the layout since no associated file path can be found.  The way I resolved this for `amphora-amp` was to instead call this `_layoutComponent`, but we could also omit the `_data._layout` field when assigning to state before passing down to the root partial.

Reply:
This might have been due to be having deprecated fields left in my page data